lib/main.dart

```diff
@@ -64,6 +66,18 @@ class SayilarState extends State<Sayilar> {
             topics: [
               // FIXME: These three topics should probably be moved into a
               // "Numbers" DirectoryTopic once more exercises are added.
+              ExerciseTopic(
+                icon: Icons.visibility,
+                title: 'Recognize time',
+                subtitle: 'on iki → *12*',
+                exercise: RecognizeTimeExercise(),
+              ),
+              ExerciseTopic(
+                icon: Icons.visibility,
+                title: 'Translate time',
+                subtitle: 'on iki → *12*',
+                exercise: TranslateTimeExercise(),
+              ),
```

Comment:
This really clutters the _"home page"_. Since there are already `DirectoryTopic`s since #27, this structure should be nested somehow.
The time questions should definitely be moved into their own directory.
But also the number questions? Or not?
Also there should probably be a `RandomExercise` (see #26) in each directory, no?